packages/app-security/src/components/SecureRoute.tsx

```diff
@@ -19,7 +31,15 @@ export default ({
 
     const plugin = getPlugin<SecureRouteErrorPlugin>("secure-route-error");
     if (!plugin) {
-        return <span>You are not authorized to view this route.</span>;
+        return (
+            <div className={notAuthCss}>    
```

Comment:
Instead of duplicating the code, please try extracting it into a reusable component.